resources/views/campaigns/roles/_table.blade.php

```diff
@@ -0,0 +1,44 @@
+@if (empty($rows) && isset($empty))
```

Comment:
This blade file shouldn't be needed as we have `views/layouts/datagrids/table.blade.php` now :)